frontend/packages/dev-console/src/components/topology/helm/TopologyHelmReleasePanel.tsx

```diff
@@ -46,6 +48,9 @@ export const ConnectedTopologyHelmReleasePanel: React.FC<TopologyHelmReleasePane
   const { manifestResources, releaseNotes } = helmRelease.getData().data;
   const name = helmRelease.getLabel();
   const { namespace } = helmRelease.getData().groupResources[0].resources.obj.metadata;
+  const selectTabQueryArg = getQueryArgument('selectTab');
+
+  if (selectTabQueryArg) removeQueryArgument('selectTab');
```

Comment:
Why not handle this generically in Topology and dispatch a redux action?

Reply:
I actually did it in my earlier implementation. But then thought this might not be useful to other side panels. Wasn't sure if we want to generally make `selectTab` available as a pre-selection API in topology.

Reply:
Updated the PR to handle this generally.